### DISCUSSION TITLE
Allow ExcalidrawAI to use local running server Ollama

### DISCUSSION THREAD

**Discussion by chiangs** [zsviczian/obsidian-excalidraw-plugin#1838] [Ideas]
I prefer to use locally running models. And in my workflow, I use the CoPilot plugin that allows me to tap into a locally running Ollama server and select different models. 

Would be great to mirror this functionality in Excalidraw.

Thanks!

**Comment by sammcj** [comment]
Came looking for this feature as well, would be great!

**Comment by chrisk-7777** [comment]
Is it documented what service is used currently for the AI side of things? I can't find it anywhere, but its possible I am overlooking it. 

I want to validate what context is being sent when using AI tooling

**Comment by Taounit** [comment]
Any update?

**Comment by imageblotter** [comment]
I'm doing my head in. Ollama should support this and when I do the calls manually they work. The settings would allow ollama support as well but i can't figure out how. Is there a way to see what data is being transferred? The console is not helpful for me.
